lib_test/test_logic.ml

```diff
@@ -41,21 +41,13 @@ module Id = struct
 
 end
 
+module DateMock = Wm_util.MakePtimeDate(struct
+  let now = fun () -> Ptime.v (17591, 0L)
```

Comment:
Any significance to the constant chosen which should be noted?

Reply:
nope, it was just the current time of writing the test. :-)